src/Relation/Nullary/Recomputable.agda

```diff
@@ -0,0 +1,39 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Recomputable types and their algebra as Harrop formulas
+------------------------------------------------------------------------
+
+{-# OPTIONS --cubical-compatible --safe #-}
+
+module Relation.Nullary.Recomputable where
+
+open import Data.Product.Base using (_×_; _,_; proj₁; proj₂)
+open import Level using (Level)
+
+private
+  variable
+    a b : Level
+    A : Set a
+    B : Set b
+
+------------------------------------------------------------------------
+-- Definition
```

Comment:
Perhaps you're right, but I personally am not that fond of modules containing at most one definition, given that the whole thing is only 40 lines long (I think you are even more extreme than me about 'small modules' and dependency thrashing...).

Reply:
So for now, I've left things as they are, plus moving the two other instances to here from `Data.Empty` and `Relation.Nullary.Negation.Core`, which may rejig the dependency graph more to your liking.

Reply:
And at least now it could easily be broken up into `Base` and `Properties` if you insisted on it... but that feels like an easy downstream refactoring...